index.js

```diff
@@ -79,6 +82,19 @@ module.exports = class NoiseSecretStream extends Duplex {
     return streamId(handshakeHash, isInitiator, id)
   }
 
+  setTimeout (ms) {
+    this._clearTimeout()
+    this._timeoutMs = ms || 0
+
+    if (!ms || this.rawStream === null) return
+    this._timeout = new Timeout(ms, destroyTimeout, this)
+  }
+
+  sendKeepAlive () {
```

Comment:
Should this one be documented?

Reply:
Ah didn't mean to commit that one! Good catch